src/analytics/private/sqls/redshift/sp-scan-metadata.sql

```diff
@@ -1,4 +1,26 @@
-CREATE OR REPLACE PROCEDURE {{schema}}.sp_scan_metadata(top_frequent_properties_limit NUMERIC, end_date DATE, start_date DATE) 
+CREATE OR REPLACE FUNCTION {{schema}}.convert_json_list(json_in varchar(65535))
```

Comment:
Let's test if it works on legacy Redshift instance type like `dc2`.

Reply:
Tested provisioned redshift dc2-instance. It works well